docs/custom-extensions/busola-web-components.md

```diff
@@ -0,0 +1,117 @@
+# Custom Busola Web Components
+
+Busola provides a suite of custom Web Components to enhance your application's functionality. These components come with utility functions to dynamically update their properties, attributes, and slots after initialization.
+
+## Utility Functions
+
+All custom Web Components expose methods to dynamically update their properties, attributes, and slots after initialization.
```

Comment:
This sentence looks almost identical to the previous one. I suggest removing the one from the Custom Busola Web Components section.